example/pythonlib/basic/1-simple/build.mill

```diff
@@ -1,34 +1,139 @@
 package build
 import mill._, pythonlib._
 
-// Docs for PythonLib
-object foo extends PythonModule {
-  object bar extends PythonModule {
-    def pythonDeps = Seq("pandas==2.2.3", "numpy==2.1.3")
-  }
-  def pythonDeps = Seq("numpy==2.1.3")
-}
+object foo extends PythonModule { f =>
+
+  def script: T[PathRef] = Task.Source { millSourcePath / "src" / "foo.py" }
```

Comment:
Let's rename this `def mainScript`, so it's a bit more clear what it is used for

Reply:
fixed!